.env.test

```diff
@@ -0,0 +1,10 @@
+# Copy this file to .env.test.local and populate with development values from Vercel dashboard
+
+BASE_URL=localhost:4000
+
+DATABASE_URL=postgresql://prisma:prisma@localhost:5433/tests
+
+SENDGRID_API_KEY= # Unfortunately this still has to be set to a real key for the client to work in sandbox mode
```

Comment:
The one issue with E2E testing is that if we are actually issuing the email using sendgrid, then we are using the sendgrid quota. If we run this in a GitHub Actions, we may use up the quota

Reply:
Good point. I have code coming in a future PR that prevents us from actually sending the email though!